server/src/app.ts

```diff
@@ -81,10 +85,50 @@ app.route("/version").get((request, response) => {
 
 // Serve React app
 app.use(isAuthenticated, serveStatic(path.join(__dirname, "../../client/build")));
+
+
 app.get("*", isAuthenticated, (request, response) => {
     response.sendFile(path.join(__dirname, "../../client/build", "index.html"));
 });
 
-app.listen(PORT, () => {
+const server = createServer(app);
```

Comment:
in docs they called this a websocket i.e. `ws` maybe we should follow, or use `wsServer`?

Reply:
that's fair but this is actually the application server (replaces `app.listen`) and SubscriptionServer is what listens for websocket messages